src/util.ts

```diff
@@ -0,0 +1,85 @@
+import BN from "bn.js";
+import { Fraction } from "./fraction";
+
+const max128 = new BN(2).pow(new BN(128)).subn(1);
+
+/**
+ * Modifies the price to work with ERC20 units
+ * @param {number} price amount of quote token in exchange for one base token
+ * @param {number} baseTokenDecimals number of decimals of the base token
+ * @param {number} quoteTokenDecimals number of decimals of the quote token
+ * @return {Fraction} fraction representing the amount of units of quote tokens in exchange for one unit of base token
```

Comment:
nit: the types are no longer needed in the comments as they are specified in the function definition. Also the `tsdoc` standard expects a `-` between the parameter name and description.

```suggestion
 * @param price - Amount of quote token in exchange for one base token
 * @param baseTokenDecimals - Number of decimals of the base token
 * @param quoteTokenDecimals - Number of decimals of the quote token
 * @return Fraction representing the amount of units of quote tokens in exchange for one unit of base token
```